adapter-base/src/test/resources/ocsp/cakey.pem

```diff
@@ -0,0 +1,6 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
I am not happy with having these certificates in the code base which seem to have *happened to just come out of nowhere*.
Can you please include a script for creating them in the `demo-certs` folder or extend the existing script and then use the keys from the demo-certs artifact as we do in other modules?

Reply:
Ok, added second device certificate to demo script and now we fetch certificates and information from that package instead of extra static files. It just required small fix, to mark certificates as end entity by CA:FALSE in constrains, otherwise the Java implementations ignores them.